src/Morpho.sol

```diff
@@ -502,11 +502,12 @@ contract Morpho is IMorphoStaticTyping {
     /// @dev Returns whether the position of `borrower` in the given market `marketParams` is healthy.
     /// @dev Assumes that the inputs `marketParams` and `id` match.
     function _isHealthy(MarketParams memory marketParams, Id id, address borrower) internal view returns (bool) {
```

Comment:
```suggestion
    function _isHealthy(MarketParams memory marketParams, Id id, address borrower) internal view returns (bool healthy) {
        healthy = true;
        
        if (position[id][borrower].borrowShares != 0) {
            uint256 collateralPrice = IOracle(marketParams.oracle).price();
            healthy = _isHealthy(marketParams, id, borrower, collateralPrice);
        }
    }
```
What about this ? Would you consider using return variables a case of an "early return" ?

Reply:
I also don't like variables that changes value in the middle of the execution :-/